queries.sql

```diff
@@ -7,4 +7,39 @@ SELECT date_of_birth FROM animals WHERE name ='piachu' OR name ='Agumon';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+
+BEGIN;
+  update animals
+  set species = 'unspecified';
+  ROLLBACK;
```

Comment:
* Kindly verify that the change was made after `species = unspecified` and verify that the species columns went back to the state before the transaction.
 ![image](https://user-images.githubusercontent.com/52497246/207435729-983980bd-510a-4292-af5a-23e563a6981a.png)